lang/en/docs/jobs/projects.md

```diff
@@ -47,6 +47,10 @@ The Project Status is indicated in [Project Explorer](ui/projects-explorer.md#st
 
 Each new Account is initialized with a default project named "Default". It is initially set to be [shared publicly](../collaboration/sharing/access-levels.md) with all platform users. Higher levels of privacy for this and all [subsequently created Projects](actions/create-delete-project.md) can be set when an appropriate [service level](../pricing/service-levels.md) is attributed to the account.
 
+## "External" Project
```

Comment:
This should be a link to here inside external upload page otherwise how users know where to look for the uploaded jobs.

Reply:
OK, there is an explanation at the end.